frontend/src/metabase/components/EmojiIcon.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint "react/prop-types": "warn" */
```

Comment:
Don't forgot to remove `frontend/src/metabase/questions/components/EmojiIcon.jsx` and replace any usage of it with this.